test-common/src/json_mock.rs

```diff
@@ -176,6 +176,24 @@ fn mock_schema_object(
             extensions: _,
         } if instance_type.as_ref() == &InstanceType::String => mock_string(format, string, src),
 
+        // UUID
+        SchemaObject {
+            metadata: _,
+            instance_type: Some(SingleOrVec::Vec(vec)),
+            format,
+            enum_values: None,
+            const_value: None,
+            subschemas: None,
+            number: _,
+            string,
+            array: _,
+            object: _,
+            reference: None,
+            extensions: _,
+        } if matches!(&vec[..], &[InstanceType::String, InstanceType::Null]) => {
```

Comment:
not sure this is strictly necessary, but for robustness:

```suggestion
        } if matches!(&vec[..], &[InstanceType::String, InstanceType::Null] | &[InstanceType::Null, InstanceType::String]) => {
```

Not required here, but I think I'll change this in the future to do something like we do in typify:

https://github.com/oxidecomputer/typify/blob/main/typify-impl/src/convert.rs#L56

Reply:
done in f393d4e